lua/nvim-tree/view.lua

```diff
@@ -352,7 +354,12 @@ function M.close()
     end
   end
   if #a.nvim_list_wins() > 1 then
-    a.nvim_win_hide(M.get_winnr())
+    local tree_win_nr = M.get_winnr()
+    local current_win_nr = a.nvim_get_current_win()
+    a.nvim_win_hide(tree_win_nr)
+    if tree_win_nr == current_win_nr and M.View.last_focus_winnr then
+      a.nvim_set_current_win(M.View.last_focus_winnr)
+    end
```

Comment:
the implementation looks correct, however, i'd like your to refactor the naming to call variables `winnr` instead of `win_nr` :)